Doc/library/stdtypes.rst

```diff
@@ -1549,6 +1549,39 @@ expression support in the :mod:`re` module).
    interpreted as in slice notation.
 
 
+.. method:: str.removeprefix(prefix, /)
+
```

Comment:
Oh, the docstring definition is short and sounds correct:

```
If the string starts with the *prefix* string, return `str[len(prefix):]`.
```

This definition implicitly documents that the prefix is removed at most once.